src/CADL.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -629,6 +637,28 @@ export function getInputType(
     }
 }
 
+export function navigateModels(
```

Comment:
Don't quite understand what this function mean, could you add more comments?

Reply:
add comment for the functionality of the method.